apps/meteor/client/views/omnichannel/directory/contacts/contextualBar/ContactInfo.js

```diff
@@ -157,13 +163,20 @@ const ContactInfo = ({ id, rid, route }) => {
 				</Margins>
 			</VerticalBar.ScrollableContent>
 			<VerticalBar.Footer>
-				<ButtonGroup stretch>
-					{showContactHistory && lastChat && (
-						<Button onClick={onChatHistory}>
+				<ButtonGroup stretch flexWrap='wrap'>
+					{isCallReady && (
+						<>
+							<VoipInfoCallButton phoneNumber={phoneNumber} mi={0} flexBasis='0' />
+							{showContactHistory && <Divider width='100%' />}
+						</>
+					)}
+
+					{showContactHistory && (
+						<Button onClick={onChatHistory} mis={0} flexBasis='0'>
```

Comment:
This logic is necessary, so we can have both layouts depending on the number of buttons:
<img width="375" alt="Screen Shot 2022-07-05 at 13 03 22" src="https://user-images.githubusercontent.com/6494543/177385334-1dc50d77-1242-43a2-8be9-f963a89d68c8.png">
<img width="375" alt="Screen Shot 2022-07-05 at 13 03 33" src="https://user-images.githubusercontent.com/6494543/177385370-3038e0c6-e990-4504-ac46-a082dc46dec4.png">

Reply:
Wait, this divider is not aligned... can you fix this?

Reply:
Yeah, this image is outdated. I reused an old print just to give the idea 😅 
Everything should be aligned.